webpack/webpack-env.d.ts

```diff
@@ -101,3 +102,137 @@ declare var __non_webpack_require__: any;
  * Equals the config option debug
  */
 declare var DEBUG: boolean;
+
+/**
+ * Webpack Hot Module Replacement
+ * https://webpack.github.io/docs/hot-module-replacement.html
+ */
+declare namespace __WebpackHotModuleReplacement {
+  interface Module {
+    exports: any;
+    require(id: string): any;
+    id: string;
+    filename: string;
+    loaded: boolean;
+    parent: any;
+    children: any[];
+    hot: Hot;
+  }
+
+  interface Hot {
+    /**
+     * Accept code updates for the specified dependencies. The callback is called when dependencies were replaced.
+     * @param dependencies
+     * @param callback
+     */
+    accept(dependencies: string[], callback: (updatedDependencies: string[]) => void): void;
+    /**
+     * Accept code updates for the specified dependencies. The callback is called when dependencies were replaced.
+     * @param dependency
+     * @param callback
+     */
+    accept(dependency: string, callback: () => void): void;
+    /**
+     * Accept code updates for this module without notification of parents.
+     * This should only be used if the module doesn’t export anything.
+     * The errHandler can be used to handle errors that occur while loading the updated module.
+     * @param errHandler
+     */
+    accept(errHandler?: Function): void;
```

Comment:
Should replace `Function` with an actual function signature. What parameters does this callback receive and what does it return?